notebooks/official/automl/sdk_automl_video_object_tracking_batch.ipynb

```diff
@@ -0,0 +1,1023 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        # Delete the AutoML or Pipeline trainig job
</pre><p>Typo: should be training</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>